modules/configuring-a-provisioning-resource-to-scale-user-provisioned-clusters.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/scaling-a-user-provisioned-cluster-with-the-bare-metal-operator.adoc
+:_content-type: PROCEDURE
+
+[id="configuring-a-provisioning-resource-to-scale-user-provisioned-clusters_{context}"]
+= Configuring a provisioning resource to scale user-provisioned clusters
+
+Create a `Provisioning` custom resource (CR) to enable Metal platform components on a user-provisioned infrastructure cluster. 
+
+.Prerequisites
+
+* You installed a user-provisioned infrastructure cluster on bare metal.
+
+.Procedure
+
+. Create a `Provisioning` CR.
+
+.. Save the following YAML in the `provisioning.yaml` file:
++
+[source,yaml]
+----
+apiVersion: metal3.io/v1alpha1
+kind: Provisioning
+metadata:
+  name: provisioning-configuration
+spec:
+  provisioningNetwork: "Disabled"
+  watchAllNamespaces: false
+----
++
+[NOTE]
+====
+{product-title} {product-version} does not support enabling a provisioning network when you scale a user-provisioned cluster by using the Bare Metal Operator.
+====
+
+. Create the `Provisioning` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f provisioning.yaml
+----
++
+.Example output
+[source,terminal]
+----
+provisioning.metal3.io/provisioning-configuration created
+----
+
+.Verification
+
+* Verify that the provisioning service is running by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-machine-api
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                          READY   STATUS    RESTARTS        AGE                             
+cluster-autoscaler-operator-79747dd75-d446d   2/2     Running   1 (2d22h ago)   2d22h                           
+cluster-baremetal-operator-547ff7f86-nnf96    2/2     Running   0               2d22h                           
+machine-api-operator-77bd9c9885-4lw7d         2/2     Running   0               2d22h                           
+metal3-6dbd9bc4db-cr8xb                       5/5     Running   0               15s                             
+metal3-image-cache-pz2sc                      1/1     Running   0               14s                             
+metal3-image-customization-86b7f889f5-w7jg9   1/1     Running   0               12s                             
```

Comment:
Dunno if it matters, but this is how the set of pods is looking for me on 4.13 (other versions may differ) when provisioning network is disabled:

```
$ oc get pod -n openshift-machine-api
NAME                                                  READY   STATUS    RESTARTS        AGE
cluster-autoscaler-operator-678c476f4c-jjdn5          2/2     Running   0               5d21h
cluster-baremetal-operator-6866f7b976-gmvgh           2/2     Running   0               5d21h
control-plane-machine-set-operator-7d8566696c-bh4jz   1/1     Running   0               5d21h
ironic-proxy-64bdw                                    1/1     Running   0               5d21h
ironic-proxy-rbggf                                    1/1     Running   0               5d21h
ironic-proxy-vj54c                                    1/1     Running   0               5d21h
machine-api-controllers-544d6849d5-tgj9l              7/7     Running   1 (5d21h ago)   5d21h
machine-api-operator-5c4ff4b86d-6fjmq                 2/2     Running   0               5d21h
metal3-6d98f84cc8-zn2mx                               5/5     Running   0               5d21h
metal3-image-customization-59d745768d-bhrp7           1/1     Running   0               5d21h
```

Reply:
Thanks @dtantsur , I copy/pasted from your very helptful demo:
![image](https://user-images.githubusercontent.com/104497497/220930318-878ed9ce-cd08-4c53-970e-4dac33452389.png)

 But I can use the updated output from this comment. Just confirming this output is from the UPI cluster?

Reply:
I think my demo was on 4.12 (or even 4.11? time flies!) before certain changes. The list is from IPI, but it should not be any different: our cluster operator does not distinguish them.